src/helper/convertColumnElementsToData.js

```diff
@@ -58,48 +100,56 @@ function convertColumnElementsToData(childComponents) {
     children.push(child);
   });
 
-  const elementTemplates = {
-    cell: [],
-    footer: [],
-    groupHeader: [],
-    header: [],
-  };
+  const columnElements = getElementsContainer();
+  const columnGroupElements = getElementsContainer();
+  const elementTemplates = getElementTemplates();
```

Comment:
just a thought shouldnt the function name be getEmptyElementContainer/ getDefaultElementContainer

or what you have done is fine?

Reply:
hmm both seem fine to me, but I'll rename it to `getEmptyElementsContainer` to be explicit.